qlib/log.py

```diff
@@ -12,7 +12,45 @@
 from .config import C
 
 
-def get_module_logger(module_name, level: Optional[int] = None):
+class MetaLogger(type):
+    def __init__(self, name, bases, dic):
+        super().__init__(name, bases, dic)
+
+    def __new__(cls, name, bases, dict):
+        wrapper_dict = type(logging.getLogger("MetaLogger")).__dict__.copy()
```

Comment:
Why not use Logger  directly?

Reply:
Will it be different by replacing  `type(logging.getLogger("MetaLogger"))` with  `logging.Logger`?
Why should we create a Logger instance  here?